src/index.html

```diff
@@ -11,7 +11,185 @@
+            <input
+              type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="date"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="agreement"
+              required
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div>
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+              minlength="8"
+              maxlength="30"
+              required
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <div>
+          <label>
+            Do you love cats?
+            <input type="radio" name="yes">
+            Yes
+            <input type="radio" name="yes">
+            No
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What is your favorite color?
+            <input type="color" name="color">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What time do you go to bed?
+            <input type="time" name="time">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What are your favorite brand of cars?
+            <select name="car" multiple>
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+            </select>
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            How do you rate our work?
+            <input
+              type="range"
+              name="range"
+              value="0"
+              min="0"
+              max="100"
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <div>
+          <label for="comm">
+            Comments:
+          </label>
+          <textarea
+            name="comments"
+            id="comm"
+            cols="20"
+            rows="2">
```

Comment:
closing bracket at same indent as opening
```suggestion
          <textarea
            name="comments"
            id="comm"
            cols="20"
            rows="2"
          >
```